frontend/src/components/SearchTool.js

```diff
@@ -215,9 +180,7 @@ const SearchTool = (props) => {
             </Col>
           </Form.Row>
           <Form.Row style={{ marginTop: "4px", marginBottom: "30px" }}>
-            <Col xs="10" style={{ paddingLeft: "15px" }}>
-              Pets allowed
-            </Col>
+            <Col xs="10">Pets allowed</Col>
```

Comment:
Did it not need the padding anymore?

Reply:
I moved the padding "up a level"- the class name `px-3` adds the padding to the parent instead of each child having their own padding